packages/web-components/fast-foundation/src/listbox/listbox.ts

```diff
@@ -343,11 +473,12 @@ export abstract class Listbox extends FoundationElement {
         prev: ListboxOption[] | undefined,
         next: ListboxOption[]
     ): void {
-        if (this.$fastController.isConnected) {
-            this.options.forEach(o => {
-                o.selected = next.includes(o);
-            });
-        }
+        const filteredNext = next.filter(Listbox.slottedOptionFilter);
+        this.options?.forEach(o => {
+            Observable.getNotifier(o).unsubscribe(this, "selected");
+            o.selected = filteredNext.includes(o);
+            Observable.getNotifier(o).subscribe(this, "selected");
```

Comment:
Interesting pattern here. I had considered an API for `Observable` like this:

```ts
Observable.stop(obj);
Observable.start(obj);

const notifier = Observable.getNotifier(obj);
notifier.isEnabled
```

Would that have been helpful in this case?

Reply:
Maybe something where you can perform a value without the notifier being called? There'd also have to be some way to configure if a notifier can be bypassed, and maybe by who/what.

```ts
const notifier = Observable.getNotifier(o);

notifier.bypass?.(this, "selected", () => {
  o.selected = filteredNext.includes(o);
});
```